src/app/content/search/hooks/index.ts

```diff
@@ -77,6 +78,7 @@ export const receiveSearchHook: ActionHookBody<typeof receiveSearchResults> = (s
       search: queryString.stringify({
```

Comment:
do we still have the concept of persistent params? right this second i can't think of any other than the search ones... but i feel like having them as a construct wouldn't be bad. you could apply them at the beginning of this object and they'd get overridden as necessary